js/extension/components/search/PlotSearch.jsx

```diff
@@ -20,6 +21,9 @@ export default function PlotsSearch({onSearch = () => {}, loading}) {
     return (
         <div className="plots-search">
             <h3><Message msgId={'cadastrapp.parcelle.title'}/></h3>
+            <div className='selectionToolsButton'>
+                <SelectionTools {...{foncier: false}}/>
```

Comment:
>  <SelectionTools {...{foncier: false}}/>

Means :

` <SelectionTools foncier="false" />`
 
 ?

Reply:
Because "foncier" is true by default in SelectionTools.jsx
And I need to make a difference between tools in main tool bar and tools in PlotSearch.

Reply:
>  <SelectionTools foncier="false" />

Is wrong syntax.

Simplification is (check with @LPoin) : 

> <SelectionTools foncier={false} />